libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -91,7 +91,14 @@ const AP_Param::GroupInfo AP_Camera::var_info[] = {
     // @Values: 0:TriggerLow,1:TriggerHigh
     // @User: Standard
     AP_GROUPINFO("FEEDBACK_POL",  9, AP_Camera, _feedback_polarity, 1),
-    
+
+    // @Param: FEEDBACK_SOURCE
+    // @DisplayName: Camera feedback location source
+    // @Description lalalala
```

Comment:
Check if this text is better:
`@Description: Normally you would not want to use any other value than the default 0, as it provides the best location estimate that the Ardupilot navigation system can get. This option exists for when you are sure you want the camera feedback location provider to be a specific GPS source, for instance a surveyor grade RTK GPS or devices with high location accuracy presenting themselves as GPS location providers.`